roles/ceph-common/tasks/installs/install_on_redhat.yml

```diff
@@ -4,7 +4,14 @@
     name: "{{ item }}"
     state: present
   with_items: redhat_package_dependencies
-  when: ansible_pkg_mgr == "yum"
```

Comment:
You probably still need this condition as we're supporting ``dnf`` below as well.